src/core/entity.js

```diff
@@ -315,7 +315,7 @@ export default class Entity extends MRElement {
     const children = Array.from(this.children);
     for (const child of children) {
       // if o is an object, traverse it again
-      if (!child instanceof Entity) {
+      if ((!child) instanceof Entity) {
```

Comment:
Is this change intentional? `!child` would be boolean (true/false) then ((!child) instanceof Entity) would be always false.

Reply:
that was leftover from a previous change, good catch - https://github.com/Volumetrics-io/mrjs/pull/521